lib/aes128.c

```diff
@@ -0,0 +1,346 @@
+/*
```

Comment:
So unfortunately we'll need to keep the same OSS license/copyright as on the other files (with Copyright belonging to the University of Michigan), without specific contributors. 
Long-term the goal is to setup a non-profit to transfer copyright ownership, and any change like this would require the approval of every contributor if we allowed copyrights for all contributors.

Reply:
Makes sense, fixed in 1e104d4.